grafana/scylla-dash-io-per-server.master.template.json

```diff
@@ -74,7 +74,10 @@
                         "transparent": false
                     },
                     {
-                        "class": "dashlist"
+                        "class": "dashlist",
+                        "tags": [
+                        	"master"
+		                ]
```

Comment:
@amnonh why do we need to update the types in the template?
I was assuming we will not have to change the template with each release